lochness/mediaflux/__init__.py

```diff
@@ -74,7 +74,11 @@ def sync_module(Lochness: 'lochness.config',
                       product: GENEActiv
                       data_dir: all_BWH_actigraphy
                       pattern: 'GENEActiv/*bin,GENEActiv/*csv'
-            
+                interviews:
+                    - product: open
+                      data_dir: PrescientXX_Interviews/OPEN
+                      out_dir: open
```

Comment:
Pronet has changed SOP for the box source directory structure. And Prescient network soon will follow the same structure.

```
ProNET
└── PronetLA
    └── PronetLA_Interviews
        ├── OPEN
        │   ├── LA11488
        │   ├── LA12256
        │   └── LA12345
        ├── PSYCHS
        │   ├── LA11488
        │   ├── LA12256
        │   └── LA12345
        └── Transcripts
            ├── Approved
            └── For_review
```

Lochness needs to use `out_dir` information in the configuration file to keep files from `OPEN` and `PSYCHS` to corresponding subdirectory for the A/V pipeline to work smoothly.
```
└── ME00077                        
    └── interviews                 
        └── open                   
            ├── OB_ME00077.m4a     
            ├── playback.m3u       
            └── zoom_0.mp4         

```